From 858d6be1ca5d9a4597e59d015e76b96705bf71df Mon Sep 17 00:00:00 2001
Message-Id: <858d6be1ca5d9a4597e59d015e76b96705bf71df.1302097026.git.jdenemar@redhat.com>
From: Eric Blake <eblake@redhat.com>
Date: Fri, 1 Apr 2011 16:18:18 -0600
Subject: [PATCH] docs: fix typo

https://bugzilla.redhat.com/show_bug.cgi?id=680190

* docs/formatdomain.html.in: Fix KVM name.
(cherry picked from commit d9588747800b4b938aad7720b559076aeebb6c4b)
---
 docs/formatdomain.html.in |    2 +-
 1 files changed, 1 insertions(+), 1 deletions(-)

diff --git a/docs/formatdomain.html.in b/docs/formatdomain.html.in
index 9b10a31..15a1f1c 100644
--- a/docs/formatdomain.html.in
+++ b/docs/formatdomain.html.in
@@ -236,7 +236,7 @@
       <dt><code>memtune</code></dt>
       <dd> The optional <code>memtune</code> element provides details
         regarding the memory tunable parameters for the domain. If this is
-        omitted, it defaults to the OS provided defaults. For QEMU/KVMi, the
+        omitted, it defaults to the OS provided defaults. For QEMU/KVM, the
         parameters are applied to the QEMU process as a whole. Thus, when
         counting them, one needs to add up guest RAM, guest video RAM, and
         some memory overhead of QEMU itself. The last piece is hard to
-- 
1.7.4.1

